src/luarocks/cmd.lua

```diff
@@ -156,15 +156,19 @@ local function get_lua_version(flags)
    if flags["lua-version"] then
       return flags["lua-version"] 
    end
-   local dirs = {
-      dir.path(cfg.home_tree, ".luarocks"),
-      cfg.sysconfdir,
-   }
+   local dirs = {}
+   local next_i = 1
    if flags["project-tree"] then
-      table.insert(dirs, 1, dir.path(flags["project-tree"], "..", ".luarocks"))
+      dirs[next_i] = dir.path(flags["project-tree"], "..", ".luarocks")
+      next_i = next_i + 1
```

Comment:
why not just use `table.insert` instead of `dirs[next_i]`? I only ask to make sure I'm not missing anything here.